packages/discovery-react-components/src/components/StructuredQuery/utils/stringifyStructuredQuerySelection.ts

```diff
@@ -19,12 +19,16 @@ const stringifyRows = (structuredQuerySelection: StructuredQuerySelection, group
     .map((rowId: number) => {
       const row = structuredQuerySelection.rows[rowId];
       const rowField = escapeReservedCharacters(row.field);
-      const rowValue = row.value === '' ? row.value : '"' + row.value + '"';
+      const rowValue = row.value === '' ? row.value : '"' + escapeDoubleQuotes(row.value) + '"';
       return `${rowField}${row.operator}${rowValue}`;
     })
     .join(structuredQuerySelection.groups[groupId].operator);
 };
 
+const escapeDoubleQuotes = (value: string) => {
+  return value.replace('"', '\\"');
```

Comment:
this will only replace the first occurrence of a double quote. if you use a regex with the `g` modifier (global), it will replace all matches